cognite/neat/rules/models/dms/_converter.py

```diff
@@ -107,7 +107,8 @@ def as_information_architect_rules(
             metadata=metadata,
             properties=SheetList[InformationProperty](data=properties),
             classes=SheetList[InformationClass](data=classes),
-            reference=self.dms.reference and self.dms.reference.as_information_architect_rules(),  # type: ignore[arg-type]
+            last=self.dms.last.as_information_architect_rules() if self.dms.last else None,
```

Comment:
I think we made a right choice of having added last, as we now have stack of three models. I imagine we can build rather strong UI lineage showing interconnection of these three.